packages/core/src/validate.js

```diff
@@ -11,13 +12,14 @@ import { isObject, mergeObjects } from "./utils";
 
 function createAjvInstance() {
   const ajv = new Ajv({
-    errorDataPath: "property",
     allErrors: true,
     multipleOfPrecision: 8,
-    schemaId: "auto",
-    unknownFormats: "ignore",
+    strictTypes: "log",
   });
 
```

Comment:
I suggest that we allow overriding ajv instance options.
```suggestion
function createAjvInstance(options) {
  const ajv = new Ajv({
    allErrors: true,
    multipleOfPrecision: 8,
    strictTypes: "log",
    ...options
  });
```

Reply:
That way we can pass the option in the form validation
```js
export default function validateFormData(
  formData,
  schema,
  customValidate,
  transformErrors,
  additionalMetaSchemas = [],
  customFormats = {}
  validationOptions={}
) {
  // Include form data with undefined values, which is required for validation.
  const rootSchema = schema;
  formData = getDefaultFormState(schema, formData, rootSchema, true);

  const newMetaSchemas = !deepEquals(formerMetaSchema, additionalMetaSchemas);
  const newFormats = !deepEquals(formerCustomFormats, customFormats);

  if (newMetaSchemas || newFormats|| validationOptions) {
    ajv = createAjvInstance(validationOptions);
  }
````

Reply:
This allows more control over validation without having to reimplement custom validate function
https://ajv.js.org/options.html